ui-validation/src/main/kotlin/edu/stanford/spezi/ui/validation/ValidationRule.kt

```diff
@@ -4,32 +4,39 @@ import edu.stanford.spezi.foundation.UUID
 import edu.stanford.spezi.ui.StringResource
 import java.util.UUID
 
-data class ValidationRule internal constructor(
-    val id: UUID = UUID(),
+enum class CascadingValidationEffect {
+    CONTINUE, INTERCEPT
+}
+
+data class ValidationRule(
     val rule: (String) -> Boolean,
     val message: StringResource,
     val effect: CascadingValidationEffect = CascadingValidationEffect.CONTINUE,
 ) {
+    companion object
+
+    // Properties
+
+    @Suppress("detekt:VariableMinLength")
+    internal val id: UUID = UUID()
```

Comment:
feel free to adapt detekt config file to exclude id from the variable min length rule

Reply:
Fine with that